_data/packages.yml

```diff
@@ -46,3 +46,7 @@
   description: 'A phenotyping pipeline for Python.'
   maintainer: ["Moritz Lürig"]
   link: "https://github.com/mluerig/phenopype"
+- package-name: pygmt
+  description: 'A Python interface for the Generic Mapping Tools.'
+  maintainer: ["Dongdong Tian", "Max Jones", "Michael Grund", "Wei Ji Leong", "Will Schlitzer"]
```

Comment:
Cc @seisman, @maxrjones, @michaelgrund, @willschlitzer to check if you're ok with having your names added here.

Reply:
thank you for this @weiji14 @seisman @maxrjones @michaelgrund @willschlitzer please let us know that you are comfortable with being listed as a maintainer here. we will then merge this PR. 

Note that not all of this information is displaying currently on the website but I will be working on that over the next months to improve package and maintainer visibility. We will also post about these packages and highlight them in the future as well.

Reply:
Fine for ne.

Reply:
Fine for me.

Reply:
That's fine for me to be listed

Reply:
thank you everyone! @weiji14 i think there are 4 thumbs up there . are you comfortable with my merging this?

Reply:
Oh yes please, go for it!

Reply:
wonderful. thank you!! merging!